data/demo-modules.yml

```diff
@@ -10,7 +10,7 @@ alpha:
     - third thing
     - final thing
   duration: 3
-
+  summative_assessment_precentage_pass_threshold: 70
```

Comment:
Would be good to trim this variable a little (47 characters is long to me)

Reply:
Trimmed this down now.